lib/core/repository/categories/cms.rb

```diff
@@ -0,0 +1,29 @@
+module Core::Repository
+  module Categories
+    class CMS < Core::Repository::Base
+      def initialize(options = {})
+        self.connection = Core::Registry::Connection[:cms]
+        self.fallback_repository = options[:fallback]
+      end
+
+      def all
+        response = connection.get('/%{locale}/categories.json' % { locale: I18n.locale })
```

Comment:
yeah cool, there are a few places that do this kind of expansion, so might be worth changing them all in one fell swoop in a separate PR.